### PR TITLE
Use annotate()ted item data (rdd or local)  in existing ml tests

### PR DESCRIPTION
**Pull request by CanBerker**
Load data through json-file and apply annotate() in existing ml tests

### CODE REVIEW

File: src/main/resources/test_files/runtime-spark/RumbleML/TransformerTests/MLTransformer-Tokenizer1.jq [hunk 1]
```diff
@@ -1,6 +1,11 @@
...
+let $data := annotate(
+    json-file("./src/main/resources/queries/rumbleML/sample-ml-data-flat.json"),
+    { "label": "integer", "name": "string", "age": "double", "weight": "double", "sentence": "string", "null": "null", "array": ["double"] }
+)
+
 let $transformer := get-transformer("Tokenizer")
 return $transformer(
-    (structured-json-file("./src/main/resources/queries/rumbleML/sample-ml-data-sentence.json")),
+    $data,
```
Comment (ghislainfourny) [inline src/main/resources/test_files/runtime-spark/RumbleML/TransformerTests/MLTransformer-Tokenizer1.jq]: Wonderful, was it a left over from before we supported RDD variables?

File: src/main/resources/test_files/runtime-spark/RumbleML/EstimatorTests/MLEstimator-LogisticRegression1.jq [hunk 2]
```diff
@@ -1,14 +1,20 @@
-(:JIQS: ShouldRun; Output="({ "label" : 0, "prediction" : 0 }, { "label" : 1, "prediction" : 1 }, { "label" : 1, "prediction" : 1 }, { "label" : 1, "prediction" : 1 }, { "label" : 1, "prediction" : 1 })" :)
+(:JIQS: ShouldRun; Output="({ "label" : 0, "prediction" : 0 }, { "label" : 1, "prediction" : 1 }, { "label" : 2, "prediction" : 2 }, { "label" : 3, "prediction" : 3 }, { "label" : 4, "prediction" : 4 }, { "label" : 5, "prediction" : 5 })" :)
+let $data := annotate(
+    json-file("./src/main/resources/queries/rumbleML/sample-ml-data-flat.json"),
+    { "label": "integer", "name": "string", "age": "double", "weight": "double", "sentence": "string", "null": "null", "array": ["double"] }
+)
+
 let $est := get-estimator("LogisticRegression")
 let $tra := $est(
```
Comment (ghislainfourny) [inline src/main/resources/test_files/runtime-spark/RumbleML/EstimatorTests/MLEstimator-LogisticRegression1.jq]: Side comment: best practice is to use complete names for variable names, for readability.

Comment (CanBerker) [reply src/main/resources/test_files/runtime-spark/RumbleML/EstimatorTests/MLEstimator-LogisticRegression1.jq]: Of course!  will try to improve these

File: src/main/resources/test_files/runtime-spark/RumbleML/TransformerTests/MLTransformer-Tokenizer1.jq (hunk 1 above)
Comment (CanBerker) [reply src/main/resources/test_files/runtime-spark/RumbleML/TransformerTests/MLTransformer-Tokenizer1.jq]: Both yes and no :) There is a good chance these tests were written before variable support indeed. But I was intentionally keeping them as they are to test as few concepts as possible in a single test. So variable lookups were intentionally left out.

**Pull request merged**